docker-compose.yml

```diff
@@ -0,0 +1,29 @@
+version: '3'
+
+services:
+  zulip:
+    build:
+      context: .
+      dockerfile: Dockerfile-dev
+    ports:
+      - "9991:9991"
+    volumes:
+      - ".:/srv/zulip"
+    command: /srv/zulip/tools/start-compose
+    depends_on:
+      - zulip-postgres
+      - zulip-memcached
+      - zulip-rabbitmq
+      - zulip-redis
+
+  zulip-postgres:
+    image: postgres:9.5
```

Comment:
For postgres, we need to install some extensions into the postgres image: `tsearch_extras` and `pgroonga`.  You can probably do that without too much trouble by calling `scripts/setup/setup-apt-repo` inside the postgres container and then using apt.

Reply:
What about the option of sewing-in postgres into zulip-core first, for now? Most of the problems are caused more by rabbitmq / memcached than postgrres.